jobhunters/routes/job.routes.js

```diff
@@ -0,0 +1,18 @@
+const jobController = require("../controllers/job.controller");
+const {authJwt} = require("../middlewares");
+
+
+module.exports = (app) => {
+    
+    app.post("/jobhunters/api/v1/jobs", [authJwt.verifyToken, authJwt.isAdminOrRecruiter], jobController.createJob);
+
+    app.put("/jobhunters/api/v1/jobs/{id}", [authJwt.verifyToken, authJwt.isAdminOrRecruiter], jobController.updateJobDetails);
```

Comment:
9 and 17 are same.. We can't have two same routes pointing to different controllers.